lib/package-manager.js

```diff
@@ -21,7 +21,11 @@ const bundledPackages = new Map([
   ['hyperclick', 'It enables alt-click for go to definition, and shift-alt-click to return to the prior location.'],
   ['go-debug', 'It allows you to interactively debug your go program and tests using delve.'],
   ['go-signature-statusbar', 'It shows function signature information in the status bar.'],
-  ['linter', 'It runs linters and displays lint results. Alternatively, you can use Facebook\'s Nuclide package instead of the linter package.']
+  ['linter', 'It runs linters and displays lint results. Alternatively, you can use atom-ide-ui.']
```

Comment:
I would prefer to replace `linter` in this list with `atom-ide-ui`. Does that simplify this somewhat?